### PR TITLE
Backend intro first draft

### PR DESCRIPTION
**Pull request by jmcgeeney**
Rough draft of backend intro

### CODE REVIEW

File: backend/1_intro_to_backend.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+At the end of this lesson, you will know the pieces that make up backend development. You won't know how they work yet, (the details are exciting, and we'll get to them later!) but you will be able to describe the functions of backend development and how it relates to API work. (Hint: they're basically the same thing) Specifically at the end of this lesson, you will be able to:
+
+* Answer the age-old question: what is an API?
+* Describe, step-by-step, how data flows from a database through a backend server and to the user.
+    * Additionally, you should know of one technology involved in each step.
+* Create a checklist of the technologies you need to learn in order to create a simple API
+
+### Specific things to learn
+
+* What does an API do?
```
[the remark this answers is not in the document]
Comment (alodahl) [reply backend/1_intro_to_backend.md]: Also, be sure to only include topics that will be covered in this outline. If they are going to learn it in others files, be explicit about it.

[the remark this answers is not in the document]
Comment (jmcgeeney) [reply backend/1_intro_to_backend.md]: Just moved! I was sort of thinking the idea behind this section would be to give a general surface-level understanding of what each piece does. The material I found mostly illustrates the concepts without going into detail. Maybe a "We will learn more about this later" comment could clarify that better?

**Comment by jmcgeeney** [comment]
Just made some updates per the feedback. Thank you!

**Comment by alodahl** [comment]
Thanks so much for working on this... I'm currently working on things in curriculum order for the new apprentices.  I'll get to this soon, I promise!

**Pull request merged**